# Main gnss antenna data acquisition cycle
gnss_hz : 50.0
# Upper and lower limits of staratus using GNSS
min_gnss_status : 4
max_gnss_status : 4
# Tolerance for the difference between the timestamps of the two antennas [sec]
time_thresshold : 0.03
# Yaw bias between the direction of travel and the vector created between the antennas
yaw_bias: 0.0